enterprise/security/src/main/java/org/neo4j/server/security/enterprise/auth/plugin/PluginRealm.java

```diff
@@ -148,8 +149,16 @@ protected AuthorizationInfo doGetAuthorizationInfo( PrincipalCollection principa
     {
         if ( authorizationPlugin != null )
         {
-            org.neo4j.server.security.enterprise.auth.plugin.spi.AuthorizationInfo authorizationInfo =
-                    authorizationPlugin.authorize( getPrincipalAndRealmCollection( principals ) );
+            org.neo4j.server.security.enterprise.auth.plugin.spi.AuthorizationInfo authorizationInfo;
+            try
+            {
+                 authorizationInfo = authorizationPlugin.authorize( getPrincipalAndRealmCollection( principals ) );
+            }
+            catch ( AuthorizationExpired e )
+            {
+                throw new AuthExpirationException(
+                        "Plugin '" + getName() + "' authorization info expired: " + e.getMessage() );
```

Comment:
The primary point of this exception is communicating a status to the client. Only the message and status code will reach the client. Nobody is looking at the stack trace so I don't see any point of attaching the cause.